lib/tasks/govuk_assets.rake

```diff
@@ -16,4 +16,27 @@ namespace :govuk_assets do
     status = result.successful? ? 'OK' : 'Error'
     puts "#{status}: #{result.written_count} documents updated"
   end
+
+  desc 'Unreplace an asset and delete the replacement'
+  task :unreplace_asset, [:uuid] => :environment do |_, args|
+    begin
+      uuid = args[:uuid]
+      asset = Asset.unscoped.find_by(uuid: uuid)
+
+      if asset.replacement.nil?
+        puts "#{uuid} has no replacement"
+        return
+      end
+
+      puts "found replacement #{asset.replacement.uuid}"
+
+      asset.replacement.deleted = true
```

Comment:
You can use `asset.replacement.update!(deleted: true)` to `update` the attributes and then call `save!`. Incidentally, it's better to use `save!` in a place like this because it will raise an exception if any of the validations fail, whereas `save` will just return `true` or `false`.